runtime/src/prioritization_fee_cache.rs

```diff
@@ -121,20 +125,26 @@ impl PrioritizationFeeCacheMetrics {
 enum CacheServiceUpdate {
     TransactionUpdate {
         slot: Slot,
+        bank_id: BankId,
         transaction_fee: u64,
         writable_accounts: Arc<Vec<Pubkey>>,
     },
     BankFrozen {
```

Comment:
come to think of it, this variant isn't about `frozen`. maybe, `BankFinalized`?

Reply:
Oh, right! fixing it.